src/vmm/src/builder.rs

```diff
@@ -273,13 +284,16 @@ pub fn build_microvm(
         (arch::IRQ_BASE, arch::IRQ_MAX),
     );
 
+    // attach boot timer as the first mmio device on either architecture
+    attach_boot_timer_device(&mut mmio_device_manager, request_ts)?;
```

Comment:
Please move this next to the other device attach functions (further down in this function).

Reply:
Done.